server/fishtest/api.py

```diff
@@ -220,9 +218,9 @@ def pgn(self):
     def worker_info(self):
         worker_info = self.request_body["worker_info"]
         worker_info["remote_addr"] = self.request.remote_addr
-        flag = self.get_flag()
-        if flag:
-            worker_info["country_code"] = flag
+        country_code = self.get_country_code()
```

Comment:
LGTM on DEV https://dfts-0.pigazzini.it/tests
I need to review/test the indexes script change before merging.

BTW this is the pythonic way (I can refactor it in the weekend)
```python
worker_info["country_code"] = (country_code := self.get_country_code()) if country_code else "?"
```

Reply:
I made it more pythoner.